src/chunk.c

```diff
@@ -4612,6 +4612,12 @@ add_foreign_table_as_chunk(Oid relid, Hypertable *parent_ht)
 	ts_chunk_constraints_add_dimension_constraints(chunk->constraints, chunk->fd.id, chunk->cube);
 	ts_chunk_constraints_insert_metadata(chunk->constraints);
 	chunk_add_inheritance(chunk, parent_ht);
+	/* Update hypertable entry with tiering status information and set overlap flag since the range
+	 * is invalid */
+	parent_ht->fd.status =
+		ts_set_flags_32(parent_ht->fd.status,
+						HYPERTABLE_STATUS_OSM | HYPERTABLE_STATUS_OSM_CHUNK_NONCONTIGUOUS);
```

Comment:
When the osm chunk is initially created there should be no data handled by osm yet and therefore we only need to set HYPERTABLE_STATUS_OSM

Reply:
done